docs/source/quickstart.rst

```diff
@@ -1,22 +1,256 @@
+        ),
+        value_map(
+            in_col="gender",
+            # Don't worry about mother/father gender here.
+            # We can create them in a later phase.
+            m={
+                "F": constants.GENDER.FEMALE,
+                "M": constants.GENDER.MALE
+            },
+            out_col=CONCEPT.PARTICIPANT.GENDER
+        ),
+        value_map(
+            in_col="specimens",
+            m=lambda x: Split(re.split("[,;]", x)),
+            out_col=CONCEPT.BIOSPECIMEN.ID
+        ),
+        [
+            value_map(
+                in_col=6,  # age (hrs) (first)
+                m=lambda x: int(x) / 24,
+                out_col=CONCEPT.PHENOTYPE.EVENT_AGE_DAYS
+            ),
+            melt_map(
+                var_name=CONCEPT.PHENOTYPE.NAME,
+                map_for_vars={
+                    "CLEFT_EGO": "Cleft ego",
+                    "CLEFT_ID": "Cleft id"
+                },
+                value_name=CONCEPT.PHENOTYPE.OBSERVED,
+                map_for_values=observed_yes_no
+            )
+        ],
+        [
+            value_map(
+                in_col=9,  # age (hrs) (second)
+                m=lambda x: int(x) / 24,
+                out_col=CONCEPT.PHENOTYPE.EVENT_AGE_DAYS
+            ),
+            melt_map(
+                var_name=CONCEPT.PHENOTYPE.NAME,
+                map_for_vars={
+                    "EXTRA_EARDRUM": "Extra eardrum"
+                },
+                value_name=CONCEPT.PHENOTYPE.OBSERVED,
+                map_for_values=observed_yes_no
+            )
+        ]
+    ]
+
+It goes in the ``extract_configs`` directory like this::
+
+    my_study/
+    ├── data
+    │   └── clinical.tsv
+    └── extract_configs
+        ├── extract_config.py
+        └── family_and_phenotype.py
+
+Run the ingest pipeline
+=======================
+
+The Warehouse Server
+--------------------
+
+Data that we ingest is meant to be shared, not stored locally on your machine!
+It's easy to configure the ingest system to connect to a remote warehousing database
+which will centrally store the relevant data.
+
+Local Environment Configuration
+-------------------------------
+
+All that's needed is the appropriate database to be setup with the required access
+permissions (see :ref:`Warehousing` for all the details). We just need to set the local variable
+``KF_WAREHOUSE_DB_URL`` with a complete login URL::
+
+    export KF_WAREHOUSE_DB_URL=postgresql://<username>:<password>@<address>:<port>/postgres
+
+The final Extraction product
+----------------------------
+
+Now we're ready to perform extraction, which is done with the following command::
+
+    $ kidsfirst ingest <your package>
+
+We're just seeing how this package works on dummy data, so we don't want
+to actually upload anything to a warehouse. This won't happen if we use the ``test``
+command, like so::
+
+    $ kidsfirst test my_study
+
+After completing this extraction that we specified using the script,
```

Comment:
I don't think we need to show the extracted table here. You already mention where ingest stage output gets written. I think thats enough